setup-tests.js

```diff
@@ -0,0 +1,2 @@
+const dotenv = require('dotenv')
+dotenv.config({path: '../.env.development'});
```

Comment:
Linked to above comment - this should already be handled in your `db/connection.js`